File: src/dataset/Java/two_fer_Schenz/gemini/GeminiFlash_Twofer_v1.java
class Twofer {
    private static final String TWOFER_DEFAULT = "One for you, one for me.";
    private static final String TWOFER_NAMED = "One for %s, one for me.";

    String twofer(String name) {
        if (name == null || name.trim().isEmpty()) {
            return TWOFER_DEFAULT;
        }

        return String.format(TWOFER_NAMED, name);
    }
}

File: src/dataset/typescript/two-fer_Exercism-typescript-shybyte/gemini/GeminiFlash_two-fer_v1.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
class TwoFer {
    static twoFer(name = TwoFer.DEFAULT_NAME) {
        return `${TwoFer.MESSAGE_START}${name}${TwoFer.MESSAGE_END}`;
    }
}
TwoFer.DEFAULT_NAME = 'you';
TwoFer.MESSAGE_START = 'One for ';
TwoFer.MESSAGE_END = ', one for me.';
exports.default = TwoFer;
